copy_this/application/commands/cacheclearcommand.php

```diff
@@ -48,12 +48,20 @@ public function help(oxIOutput $oOutput)
     public function execute(oxIOutput $oOutput)
     {
         $oInput = $this->getInput();
+
+        $oOutput->writeLn('Clearing OXID db and backend cache..');
+        /** @var oxCache $oCache */
+        $oCache = oxNew('oxcache');
```

Comment:
`oxCache` does not exist in community edition, therefore it will fail.
Also I'm willing to release patches for both 1.1 and 1.2 versions with this change, but I want to keep you as original contributor of that. So would you mind making a new pull request with just solving this particular problem?